packages/ui/src/components/logo/logo-name-map.ts

```diff
@@ -0,0 +1,9 @@
+import { siBitbucket, siGithub, siGitlab, siJira, siKubernetes, type SimpleIcon } from 'simple-icons'
+
+export const LogoNameMap = {
+  github: siGithub,
+  gitlab: siGitlab,
+  bitbucket: siBitbucket,
+  jira: siJira,
+  kubernetes: siKubernetes
+} satisfies Record<string, SimpleIcon>
```

Comment:
Not a problem, but just curious why you opted for `satisfies` over typing the variable? Don't feel the need to change it, I'm just wondering that's all.
```typescript tsx
export const LogoNameMap: Record<string, SimpleIcon> = {
  github: siGithub,
  gitlab: siGitlab,
  bitbucket: siBitbucket,
  jira: siJira,
  kubernetes: siKubernetes
}
```

Reply:
I agree using the type itself is better here. I've cleaned it up.